.github/workflows/deploy-blog-in-prod.yaml

```
@@ -1,10 +1,14 @@
-name: Build the blog image and deploy
+name: Build Prod Image
```

I think the deploy bit is still relevant.

Maybe you want something like
```suggestion
name: Build and deploy Prod Image
```
?